torch/_inductor/decomposition.py

```diff
@@ -173,6 +173,9 @@ def special_erf(x):
     a5 = 1.061405429
     p = 0.3275911
 
+    if x.type == torch.bool:
+        x = torch.gt(x)
```

Comment:
Doesn't `torch.gt` take two args?  What does this line do?

Reply:
my bad! I incorrectly copied from my previous PR to torchdynamo.

Reply:
Just fixed it. Changing the - operator to ~ when x is a bool tensor doesn't produce the correct output, so I converted it to an int tensor.